views/release.pug

```diff
@@ -26,6 +26,7 @@ html
               p(class="release-cdTitle") SECONDS
               div(class="release-cdTimer"  id="release-timerSeconds") XX
 
+    script(src="../static/js/releaseDayJS.js")
```

Comment:
Change this into "/static/js/releaseDay.js", so the path *and* remove the "JS" part of the filename.